docs/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Next
 
+* Integrating with vendorengrams.xyz to find at-level vendor drops.
```

Comment:
Please always put an empty line between the changes and the next heading. The auto-scraper gets confused.

Reply:
My bad - I think that was an auto-merge casualty.